src/handle/mod.rs

```diff
@@ -181,6 +181,12 @@ impl<'a, T: Value> Handle<'a, T> {
         self.downcast(cx).or_throw(cx)
     }
 
+    #[cfg(feature = "napi-1")]
+    pub fn strict_equals<'b, U: Value, C: Context<'b>>(&self, cx: &mut C, other: Handle<'b, U>) -> bool {
```

Comment:
Why is this on `Handle`? Shouldn't this be on `trait Value`?

Reply:
Discussed offline, and we came to this thought:
- It's nice to have it by default without having to import `Value`
- There's not much on `Value`
- Maybe in the future we could consider adding most/all of the `Value` methods on `Handle` and vice versa, so there isn't such a confusing split. (In an ambiguity, rustc will always let the inherent methods take precedence, so as long as the methods are consistent, there is no harm in the duplication.)